extensions/blocks/eventbrite/editor.scss

```diff
@@ -7,9 +7,16 @@
 
 .wp-block-jetpack-eventbrite.is-modal-button {
 	display: inline-block;
+
 	.wp-block-button__link {
 		display: block;
 	}
+
+	.wp-block-button__link:focus {
+		span[data-rich-text-placeholder]:after {
+			opacity: 0.3;
```

Comment:
Is there a reason for this?
Core Guten sets it at 0.8 for accessibility/contrast reasons:

https://github.com/WordPress/gutenberg/blob/164d5830a9acd895dfd661b5fde1ca8b80b270ac/packages/block-library/src/button/editor.scss#L35-L38

Reply:
This is specifically for when the user selects the button, it fades out the placeholder label to better indicate that the placeholder can be typed over and isn't a value.

Reply:
Ah got it, I missed the `:focus` there.